Tests/Controller/AbstractConnectControllerTest.php

```diff
@@ -20,7 +20,7 @@
 use HWI\Bundle\OAuthBundle\Security\OAuthUtils;
 use HWI\Bundle\OAuthBundle\Tests\Fixtures\CustomOAuthToken;
 use HWI\Bundle\OAuthBundle\Tests\Fixtures\CustomUserResponse;
-use Symfony\Bundle\FrameworkBundle\Tests\TestCase;
+use PHPUnit\Framework\TestCase;
```

Comment:
This is already fixed, can you rebase your PR please? :)

Reply:
Yes, updated. thanks.